website/source/docs/drivers/external/firecracker-task-driver.html.md

```diff
@@ -0,0 +1,110 @@
+   Defaults to 'vmlinux' on nomad's allocation directory.
+
+* `BootDisk` - (Optional) Path to the ext4 rootfs to boot from.  
+   Defaults to 'rootfs.ext4' on nomad's allocation directory. 
+
+* `BootOptions` - (Optional) Kernel command line options to boot the microvm.  
+   Defaults to "ro console=ttyS0 reboot=k panic=1 pci=off"
+
+* `Network` - (Optional) Network name of your container network configuration file.
+ 
+* `Vcpus` - (Optional)  Number of cpus to assign to microvm.
+
+* `Cputype` - (Optional) Cpu template to use, templates available are C3 or T2.
+
+* `Mem` - (Optional) Amount of memory in Megabytes to assign to microvm.
+   Defaults to 512
+
+* `Firecracker` Location of the firecracker binary, the option could be omitted if the environment
+   variable FIRECRACKER_BIN is setup.
+   Defaults to '/usr/bin/firecracker'  
+
+* `DisableHt` - (Optional) Disable CPU Hyperthreading
+   Defaults to false 
+
+* `Log` - (Optional) path to file where to write firecracker logs.
+
+## Networking
+
+Network configuration is setup using CNI plugins, the steps to setup firecracker task driver with cni are the following:
+
+-    Build [cni plugins][Container network plugins] and [tc-redirect-tap][tc-redirect-tap] and copy them /opt/cni
+-    Create a network configuration to be used by micro-vms on /etc/cni/conf.d/, for example: default.conflist.
+
+### Example network configuration
+```hcl
+{
+	"name": "default",
+		"cniVersion": "0.4.0",
+		"plugins": [
+		{
+			"type": "ptp",
+			"ipMasq": true,
+			"ipam": {
+				"type": "host-local",
+				"subnet": "192.168.127.0/24",
+				"resolvConf": "/etc/resolv.conf"
+			}
+		},
+		{
+			"type": "firewall"
+		},
+		{
+			"type": "tc-redirect-tap"
+		}
+		]
+}
+```
+In this example the name of this network is default and this name is the parameter used in Network on the task driver job spec. Also the filename must match the name of the network, and the suffix .conflist.
+
+## Client Requirements
+
+`firecracker-task-driver` requires the following:
+
+* Linux 4.14+ Firecracker currently supports physical Linux x86_64 and aarch64 hosts, running kernel version 4.14 or later. However, the aarch64 support is not feature complete (alpha stage).
+* The [Firecracker binary][Firecracker binary]
+* KVM enabled in your Linux kernel, and you have read/write access to /dev/kvm.
+* tun kernel module.
+* The firecracker-task-driver binary placed in the [plugin_dir][plugin_dir] directory. 
+* ip6tables package
+* [Container network plugins][Container network plugins]
+* [tc-redirect-tap][tc-redirect-tap] 
+
+[plugin_dir]: /docs/configuration/index.html#plugin_dir
+[tc-redirect-tap]:https://github.com/firecracker-microvm/firecracker-go-sdk/tree/master/cni
+[Container network plugins]:https://github.com/containernetworking/plugins 
+[Firecracker binary]:https://github.com/firecracker-microvm/firecracker/releases/download/v0.16.0/firecracker-v0.16.0
```

Comment:
Let's change this link to point to the latest release page, rather than directly to the binary for a specific version. That gives folks an opportunity to check the SHA/signatures.

Reply:
Done!.